docs/insomnia/install.md

```diff
@@ -44,22 +54,26 @@ sudo apt-get update
 sudo apt-get install insomnia
 ```
 
-### Other Linux Distributions
+You can also download the [latest Debian package](https://download.konghq.com/insomnia-ubuntu/).
+
+### Snap
 
-Snap is a new cross-platform package format that supports convenient auto-updates. You can view Insomnia on Snapcraft or install it directly with the following command.
+[Snap](https://snapcraft.io/) is a new cross-platform package format that supports convenient auto-updates. You can view [Insomnia on Snapcraft](https://snapcraft.io/insomnia) or install it directly with the following command.
 
-`sudo snap install insomnia`
+```bash
+sudo snap install insomnia
+```
 
-There is also a portable AppImage package that can be run directly as an executable. Double click the package or run it from the command line.
+There's also a [portable AppImage package](https://github.com/Kong/insomnia/releases/tag/core%402021.6.0) that can be run directly as an executable.
```

Comment:
This links to the particular release and the link will become outdated when we push a new release 🤔 making a note of this in INS-1169, you can leave as-is in this PR.